<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mmmm">
   <widget class="QTabWidget" name="mmm">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="reclamation_2">
     <attribute name="title">
      <string>reclamation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>441</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
border-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="title">
       <string>ajouter</string>
      </property>
      <widget class="QLineEdit" name="Num_avis">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(255, 170, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Nirmala UI</family>
         <pointsize>8</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 255, 255);

border-color: rgb(121, 121, 121);
font: 8pt &quot;Orator Std&quot;;
font: 75 8pt &quot;Nirmala UI&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;i&gt;Num_avis
&lt;/i&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;i&gt;Nom
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Nom">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;i&gt;Objet&lt;i&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Objet">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;i&gt;Reclamation&lt;i&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="annuler">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0) ;</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Reclamation">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(40, 40, 40);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>270</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <zorder>Nom</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>Objet</zorder>
      <zorder>label_6</zorder>
      <zorder>annuler</zorder>
      <zorder>Reclamation</zorder>
      <zorder>Num_avis</zorder>
      <zorder>ajouter</zorder>
      <zorder>modifier</zorder>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>451</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="title">
       <string>afficher </string>
      </property>
      <widget class="QPushButton" name="tri_nom">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>trier(objet)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_cin">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>300</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>trier(reclamtion)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_pass">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>300</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>trier(nom)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actualiser">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>actualiser</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>371</width>
         <height>80</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>recherche</string>
       </property>
       <widget class="QLineEdit" name="rech">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;i&gt;Entrer Num_avis&lt;i&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tab_c">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>291</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_3">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>Actualiser</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_4">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>200</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_nom_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>QRCode</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>821</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color :rgb(255, 255, 255)</string>
      </property>
      <property name="title">
       <string>supprimer </string>
      </property>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin_sup">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>&lt;i&gt;Num_avis&lt;i&gt;</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="hlkjl">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
